h2/src/main/org/h2/store/fs/retry/FileRetryOnInterrupt.java

```diff
@@ -82,17 +82,6 @@ public void implCloseChannel() throws IOException {
         }
     }
 
-    @Override
-    public long position() throws IOException {
-        for (int i = 0;; i++) {
-            try {
-                return channel.position();
-            } catch (IOException e) {
-                reopen(i, e);
```

Comment:
Not sure, if it is possible at all to be interrupted and get ClosedChannelException during this operation,  but that logic is lost now, isn't it?